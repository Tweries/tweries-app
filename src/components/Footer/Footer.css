/* BEM */

footer {
  align-items: center;
  color: #721c24;
  display: flex;
  flex-direction: column;
  font-size: 12px;
  margin-bottom: 8px;
}

footer.healthy {
  color: #155724;
}

footer a {
  color: rgb(29, 161, 242);
  font-weight: bold;
  text-decoration: none;
}

footer ul {
  display: flex;
}

footer li {
  margin: 0 8px;
}

footer p {
  margin: 0 0 4px 0;
}
