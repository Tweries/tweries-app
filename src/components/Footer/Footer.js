import classnames from 'classnames';
import React from 'react';
import './Footer.css';

function Footer({ healthy, version }) {
  return (
    <footer className={classnames({ healthy: healthy })}>
      <p>
        Tweries v{version} &copy; {new Date().getFullYear()}{' '}
      </p>
      <ul>
        <li>
          <a
            href="https://twitter.com/TweriesApp"
            rel="noopener noreferrer"
            target="_blank"
          >
            @TweriesApp
          </a>
        </li>
        <li>
          <a
            href="https://github.com/Tweries"
            rel="noopener noreferrer"
            target="_blank"
          >
            github.com/Tweries
          </a>
        </li>
      </ul>
    </footer>
  );
}

export default Footer;
