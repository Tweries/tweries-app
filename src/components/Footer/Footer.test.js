import { render } from '@testing-library/react';
import React from 'react';
import Footer from './Footer';

describe('Footer', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Footer healthy={true} version="1.0.0-beta.1" />
    );
    expect(container).toMatchSnapshot();
  });
});
