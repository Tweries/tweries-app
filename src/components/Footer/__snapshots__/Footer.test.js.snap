// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer to match snapshot 1`] = `
<div>
  <footer
    class="healthy"
  >
    <p>
      Tweries v
      1.0.0-beta.1
       Â© 
      2019
       
    </p>
    <ul>
      <li>
        <a
          href="https://twitter.com/TweriesApp"
          rel="noopener noreferrer"
          target="_blank"
        >
          @TweriesApp
        </a>
      </li>
      <li>
        <a
          href="https://github.com/Tweries"
          rel="noopener noreferrer"
          target="_blank"
        >
          github.com/Tweries
        </a>
      </li>
    </ul>
  </footer>
</div>
`;
